### ISSUE TITLE
[luci/pass] Extend FuseInstanceNormPass with keras support

### ISSUE THREAD

**Issue by mbencer** [Samsung/ONE#16347]
### Motivation

The are few combinations of `InstaneNorm` pattern in decomposed version. One of them especially important for TF is:
```
tf.contrib.layers.instance_norm
```
Unfortunately, it requires pretty old version of TF - 2.6.5 (not available for the newer versions of Python).

### Solution

Based on the specification a good replacement is `tf.keras.layers.GroupNormalization` with `groups` argument set to `-1`.
In https://www.tensorflow.org/api_docs/python/tf/keras/layers/GroupNormalization we can find:

> Relation to Instance Normalization: If the number of groups is set to the input dimension (number of groups is equal to number of channels), then this operation becomes identical to Instance Normalization. You can achieve this via groups=-1.

Some other analyzed candidate was `tf.keras.layers.LayerNormalization` but it doesn't handle `gamma` and `beta` arguments in the way expected by ONE (not in the per-channel way).

### The test model:
- tf/kreas:
```
...
gamma_initializer = tf.keras.initializers.RandomNormal(mean=0, stddev=0.1)
beta_initializer = tf.keras.initializers.RandomNormal(mean=0, stddev=0.1)

in_ = tf.compat.v1.placeholder(dtype=tf.float32, shape=(1, 16, 16, 32), name="Hole")
layer = tf.keras.layers.GroupNormalization(groups=-1, epsilon=1e-6, center=True, scale=True, gamma_initializer=gamma_initializer, beta_initializer=beta_initializer)
...
```
- tflite (note that the pair of reshapes on the beginning and the end is added):

<img width="395" height="942" alt="Image" src="https://github.com/user-attachments/assets/3dcb4f05-d14e-4bb7-9132-7125e68bb6de" />

- circle:

<img width="428" height="930" alt="Image" src="https://github.com/user-attachments/assets/0f7c0b16-64c9-464a-a8a5-3b039e37666d" />
 
- .opt.circle:

<img width="200" height="300" alt="Image" src="https://github.com/user-attachments/assets/040eafc8-656f-45cb-8042-5bdf55f9ae9d" />

.tflite, .circle, .opt.circle files: [instance_norm.zip](https://github.com/user-attachments/files/24674756/instance_norm.zip)

**Note that the pass works on the model produced by the current supported version of `tensorflow ==2.18.1` (https://github.com/Samsung/ONE/pull/16114) which can be changed.**
